generative_data_prep/utils/large_file_shuffle.py

```diff
@@ -20,6 +20,8 @@
 
 from tqdm import tqdm
 
+from .logger import logger
```

Comment:
import `Logger` instead of `logger`

Reply:
actually nvm I take this back, your way is better

Reply:
sorry, not sure actually, need to weigh the pros and cons of Singleton vs Global Variable